test/unit/quality-labels-test.js

```diff
@@ -0,0 +1,99 @@
+define([
+    'utils/quality-labels',
+    'utils/underscore',
+], function (qualityLabels, _) {
+    QUnit.module('quality-labels');
+    var test = QUnit.test.bind(QUnit);
+
+    var customLabels  = {
+        1000: 'low',
+        2000: 'medium',
+        3000: 'high',
+    };
+
+    var bandwidths = _.keys(customLabels);
+
+    test('creates a label using height when available', function (assert) {
+        var expected = '360p';
+        var actual = qualityLabels.createLabel(360, undefined, false);
+        assert.equal(actual, expected);
+    });
+
+    test('creates a label using bandwidth when height is not available', function (assert) {
+        var expected = '1337 kbps';
+        var actual = qualityLabels.createLabel(undefined, 1337000,  false);
+        assert.equal(actual, expected);
+    })
+
```

Comment:
Add a test with args `undefined, 1337000, true` to verify that parentheses are not added.